apps/condo/domains/billing/schema/BillingReceipt.test.js

```diff
@@ -735,6 +735,105 @@ describe('BillingReceipt', () => {
+                    const [receipt, attrs] = await createTestBillingReceipt(admin, context, property, account)
+
+                    expect(attrs).not.toHaveProperty('receiver')
+                    expect(attrs).toHaveProperty(['recipient', 'tin'])
+                    expect(receipt).toHaveProperty(['recipient', 'tin'])
+                    expect(receipt).toHaveProperty(['receiver', 'tin'], receipt.recipient.tin)
+                })
+                test('Should create recipient and set receiver.isApproved field automatically if billing integration is trusted and tin matches', async () => {
+                    const {
+                        context: trustedBillingContext,
+                        organization: trustedOrganization,
+                    } = await makeContextWithOrganizationAndIntegrationAsAdmin({ isTrustedBankAccountSource: true })
+                    const tin = trustedOrganization.tin
+                    const [trustedProperty] = await createTestBillingProperty(admin, trustedBillingContext)
+                    const [trustedAccount] = await createTestBillingAccount(admin, trustedBillingContext, trustedProperty)
+                    const [receipt, attrs] = await createTestBillingReceipt(admin, trustedBillingContext, trustedProperty, trustedAccount, { recipient: createTestRecipient({ tin: tin }) })
+
+                    expect(attrs).not.toHaveProperty('receiver')
+                    expect(attrs).toHaveProperty(['recipient', 'tin'])
+                    expect(receipt).toHaveProperty(['recipient', 'tin'])
+                    expect(receipt).toHaveProperty(['receiver', 'tin'], receipt.recipient.tin)
+                    expect(receipt).toHaveProperty(['receiver', 'isApproved'], true)
+                })
+                test('Should create recipient and not set receiver.isApproved field automatically if billing integration is not trusted and tin matches', async () => {
+                    const {
+                        context: trustedBillingContext,
+                        organization: trustedOrganization,
+                    } = await makeContextWithOrganizationAndIntegrationAsAdmin({ isTrustedBankAccountSource: false })
+                    const tin = trustedOrganization.tin
+                    const [trustedProperty] = await createTestBillingProperty(admin, trustedBillingContext)
+                    const [trustedAccount] = await createTestBillingAccount(admin, trustedBillingContext, trustedProperty)
+                    const [receipt, attrs] = await createTestBillingReceipt(admin, trustedBillingContext, trustedProperty, trustedAccount, { recipient: createTestRecipient({ tin: tin }) })
+
+                    expect(attrs).not.toHaveProperty('receiver')
+                    expect(attrs).toHaveProperty(['recipient', 'tin'])
+                    expect(receipt).toHaveProperty(['recipient', 'tin'])
+                    expect(receipt).toHaveProperty(['receiver', 'tin'], receipt.recipient.tin)
```

Comment:
```suggestion
                    expect(receipt).toHaveProperty(['receiver', 'tin'], receipt.recipient.tin)
```

Reply:
This tests will be removed soon. As hook in BillingReceipt will be deleted